src/server.ts

```diff
@@ -1 +1,19 @@
-console.log('Hello Nightmare King');
+// console.log('Hello Nightmare King');
+
+// import fs from 'fs';
+
+// fs.readFile('src/db.json', 'utf-8', (err, data) => {
+//   if (err) {
+//     console.error(err);
+//     return;
+//   }
+//   console.log(data);
+// });
+
+import { readFile } from 'fs/promises';
+
+async function readPasswordFile() {
+  const data = await readFile('src/db.json', 'utf-8');
+  console.log(data);
+}
+readPasswordFile();
```

Comment:
I'd prefer these changes to not be included, since we've moved this functionality to `src/utils/credentials.ts`
I recommend reverting this with another commit. We'll work on our server in the next exercise

Reply:
Will do, thank you